lib/SILGen/SILGenExpr.cpp

```diff
@@ -1693,7 +1693,7 @@ RValue RValueEmitter::visitMemberRefExpr(MemberRefExpr *E, SGFContext C) {
   // and releases.
   if (auto *N = E->getBase()->getType()->getNominalOrBoundGenericNominal())
     if (auto RV = NominalTypeMemberRefRValueEmitter(E, C, N).emit(SGF))
-      return RValue(std::move(RV).getValue());
+      return RValue(const_cast<RValue&&>(std::move(RV).getValue()));
```

Comment:
Hmm, seems like a Visual Studio bug, rather than something MSVC specific. I can reproduce this from the IDE in intellisense but not MSVC. I wrote this in about a month ago so I think I just assumed IDE errors == MSVC errors for this.

The error is:
```
swift::lowering::RValue(const swift::Lowering::RValue &) is inaccessible
```